tests/Dialect/Field/cubic_ext_field_runner.mlir

```diff
@@ -0,0 +1,140 @@
+  %add0_i32 = field.bitcast %add0 : !PF -> i32
+  %add1_i32 = field.bitcast %add1 : !PF -> i32
+  %add2_i32 = field.bitcast %add2 : !PF -> i32
+  %add_tensor = tensor.from_elements %add0_i32, %add1_i32, %add2_i32 : tensor<3xi32>
+  %add_memref = bufferization.to_buffer %add_tensor : tensor<3xi32> to memref<3xi32>
+  %add_unranked = memref.cast %add_memref : memref<3xi32> to memref<*xi32>
+  func.call @printMemrefI32(%add_unranked) : (memref<*xi32>) -> ()
+
+  // Test 2: Subtraction - a - b = (-1, -1, 2) = (6, 6, 2) in F₇
+  %sub_result = field.sub %a, %b : !CF
+  %sub0, %sub1, %sub2 = field.ext_to_coeffs %sub_result : (!CF) -> (!PF, !PF, !PF)
+  %sub0_i32 = field.bitcast %sub0 : !PF -> i32
+  %sub1_i32 = field.bitcast %sub1 : !PF -> i32
+  %sub2_i32 = field.bitcast %sub2 : !PF -> i32
+  %sub_tensor = tensor.from_elements %sub0_i32, %sub1_i32, %sub2_i32 : tensor<3xi32>
+  %sub_memref = bufferization.to_buffer %sub_tensor : tensor<3xi32> to memref<3xi32>
+  %sub_unranked = memref.cast %sub_memref : memref<3xi32> to memref<*xi32>
+  func.call @printMemrefI32(%sub_unranked) : (memref<*xi32>) -> ()
+
+  // Test 3: Multiplication - a * b (with ξ = 2, v³ = 2)
+  %mul_result = field.mul %a, %b : !CF
+  %mul0, %mul1, %mul2 = field.ext_to_coeffs %mul_result : (!CF) -> (!PF, !PF, !PF)
+  %mul0_i32 = field.bitcast %mul0 : !PF -> i32
+  %mul1_i32 = field.bitcast %mul1 : !PF -> i32
+  %mul2_i32 = field.bitcast %mul2 : !PF -> i32
+  %mul_tensor = tensor.from_elements %mul0_i32, %mul1_i32, %mul2_i32 : tensor<3xi32>
+  %mul_memref = bufferization.to_buffer %mul_tensor : tensor<3xi32> to memref<3xi32>
+  %mul_unranked = memref.cast %mul_memref : memref<3xi32> to memref<*xi32>
+  func.call @printMemrefI32(%mul_unranked) : (memref<*xi32>) -> ()
+
+  // Test 4: Square - a²
+  %square_result = field.square %a : !CF
+  %sq0, %sq1, %sq2 = field.ext_to_coeffs %square_result : (!CF) -> (!PF, !PF, !PF)
+  %sq0_i32 = field.bitcast %sq0 : !PF -> i32
+  %sq1_i32 = field.bitcast %sq1 : !PF -> i32
+  %sq2_i32 = field.bitcast %sq2 : !PF -> i32
+  %sq_tensor = tensor.from_elements %sq0_i32, %sq1_i32, %sq2_i32 : tensor<3xi32>
+  %sq_memref = bufferization.to_buffer %sq_tensor : tensor<3xi32> to memref<3xi32>
+  %sq_unranked = memref.cast %sq_memref : memref<3xi32> to memref<*xi32>
+  func.call @printMemrefI32(%sq_unranked) : (memref<*xi32>) -> ()
+
+  // Test 5: Inverse - a⁻¹
+  %inv_result = field.inverse %a : !CF
+  %inv0, %inv1, %inv2 = field.ext_to_coeffs %inv_result : (!CF) -> (!PF, !PF, !PF)
+  %inv0_i32 = field.bitcast %inv0 : !PF -> i32
+  %inv1_i32 = field.bitcast %inv1 : !PF -> i32
+  %inv2_i32 = field.bitcast %inv2 : !PF -> i32
+  %inv_tensor = tensor.from_elements %inv0_i32, %inv1_i32, %inv2_i32 : tensor<3xi32>
+  %inv_memref = bufferization.to_buffer %inv_tensor : tensor<3xi32> to memref<3xi32>
+  %inv_unranked = memref.cast %inv_memref : memref<3xi32> to memref<*xi32>
+  func.call @printMemrefI32(%inv_unranked) : (memref<*xi32>) -> ()
```

Comment:
instead of this what about printing a times a^-1, since hard to check whether the value is correct.